exonum-java-binding-time-service/src/test/java/com/exonum/binding/time/TimeSchemaProxyIntegrationTest.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2019 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.exonum.binding.time;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import com.exonum.binding.proxy.Cleaner;
+import com.exonum.binding.proxy.CloseFailuresException;
+import com.exonum.binding.storage.database.MemoryDb;
+import com.exonum.binding.storage.database.Snapshot;
+import com.exonum.binding.test.RequiresNativeLibrary;
+import com.exonum.binding.util.LibraryLoader;
+import java.util.function.Consumer;
+import org.junit.jupiter.api.Test;
+
+@RequiresNativeLibrary
+class TimeSchemaProxyIntegrationTest {
+
+  static {
+    LibraryLoader.load();
+  }
+
+  @Test
+  void getTimeTest() {
+    assertSchema((schema) -> assertFalse(schema.getTime().isPresent()));
+  }
+
+  @Test
+  void getValidatorsTimesTest() {
+    assertSchema((schema) -> assertFalse(schema.getValidatorsTimes().entries().hasNext()));
+  }
+
+  private static void assertSchema(Consumer<TimeSchemaProxy> assertion) {
+    try (MemoryDb db = MemoryDb.newInstance(); Cleaner cleaner = new Cleaner()) {
```

Comment:
Please do not put several statements in a single line: https://google.github.io/styleguide/javaguide.html#s4.3-one-statement-per-line